docker-compose.override.yml

```diff
@@ -0,0 +1,11 @@
+version: '3'
+
+services:
+    assets-builder:
+        volumes:
+            - ./:/srv/pattern-library
+            - /srv/pattern-library/node_modules
```

Comment:
This doesn't think what we/I thought it does: it creates a volume for `node_modules`, and like most (all?) Docker volumes it copies the contents of that folder from the image into the volume. This being a builder container, it doesn't matter very much as downstream Dockerfiles will copy files from the `assets-builder` image, not from any volume. We also create a container in `bin/gulp-watch`, but it's a throwaway that gets re-created every time is needed (I should add `-v` to `docker rm`). But if it's a service it may cause problems as the volume is not updated when a new image is run...

Reply:
Ah, https://stackoverflow.com/a/38601156/77199 mentions persistence.